tools/packaging/packaging.mk

```diff
@@ -1,8 +1,195 @@
 #remove leading characters since package version expects to start with digit
 PACKAGE_VERSION ?= $(shell echo $(VERSION) | sed 's/^[a-z]*-//' | sed 's/-//')
 
-# Building the debian file, docker.istio.deb and istio.deb
-include tools/packaging/deb/istio.mk
+# Creates the 2 packages. BUILD_WITH_CONTAINER=1 or in CI/CD (BUILD_WITH_CONTAINER=0)
+#
+# Development/manual testing:
+#    make deb          - builds debian packaging
+#    make deb/docker   - builds a test docker image
 
-# RPM/RHEL/CENTOS stuff
-include tools/packaging/rpm/rpm.mk
+deb: ${ISTIO_OUT_LINUX}/release/istio-sidecar.deb ${ISTIO_OUT_LINUX}/release/istio.deb
```

Comment:
Is this copied verbatim from the previous deb/istio.mk ?
I have a merge conflict now with that file now.

Reply:
Yes should be the same